.github/workflows/stale.yml

```
@@ -0,0 +1,28 @@
+# https://github.com/actions/stale
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+concurrency:
+  group: lock
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
```

Should we start like this? These are the most offending ones IMO, and then if it works we can do some cleanup and expand to more issues/PRs.

```suggestion
        with:
          any-of-labels: need info,Waiting for response
```

---

Okay. I wanted to go with `exempt-*` but I guess starting with a white list of what to mark stale works too 😊